packages/hawtio/src/plugins/camel/help.md

```diff
@@ -0,0 +1,15 @@
+### Camel
```

Comment:
The top section level for a help should be `##` (h2), because the topmost level h1 is reserved for the title *Help* in the page. Check the other helps for reference.